proto/utxorpc/v1alpha/query/query.proto

```diff
@@ -24,7 +24,41 @@ message ReadParamsRequest {
   google.protobuf.FieldMask field_mask = 1; // Field mask to selectively return fields in the parsed response.
 }
 
-// An evenlope that holds parameter data from any of the compatible chains
+// Request to get the summary of eras
+message ReadEraSummaryRequest {
+}
+
+// Response containing the summary of eras
+message ReadEraSummaryResponse {
+  oneof summary {
+    utxorpc.v1alpha.cardano.EraSummaries cardano = 1; // Cardano era summary
+  }
+}
+
+// Request to get the configuration information for an era
+message ReadEraConfigRequest {
+  string name = 1; // name of the era
+}
+
+// Response containing the summary of eras
+message ReadEraConfigResponse {
+  oneof summary {
+    utxorpc.v1alpha.cardano.EraConfig cardano = 1; // Cardano era config
+  }
+}
+
+// Request to get the chain ID
+message ReadChainIdRequest {
+}
+
+// Response containing the summary of eras
+message ReadChainIdResponse {
+  bytes genesis = 1; // genesis hash for the chain
+  string caip2 = 2; // the caip-2 ID for this network
```

Comment:
CAIP2 is a chain-agnostic way of specifying networks for blockchains. You can see the spec [here](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md)

Notably, the caip2 notation for Cardano is defined by [cip34](https://cips.cardano.org/cip/CIP-34)

Reply:
I wonder if we should add a "name" field

Tentatively, I think not because nodes like Cardano don't expose a "name" field so it would probably end up being annoying to deal with